content/docs/operating/configuration.md

```diff
@@ -239,7 +243,7 @@ Where `<filename_pattern>` may be a path ending in `.json`, `.yml` or `.yaml`. T
 may contain a single `*` that matches any character sequence, e.g. `my/path/tg_*.json`.
 
 
-### Relabeling `<relabel_config>`
+### Target Relabeling `<relabel_config>`
```

Comment:
We don't do title case (unfortunately). This also apply to the uppercase Rs below.

We could make it uppercase globally though as it is more of a custom concept name and not self-explanatory. This would require changes to other parts of the docs as well.